src/stories/6_Addons.stories.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Overview/Addons" />
+
+# Addons
+
+The RoleModel Design System provides a few addons for integrating Third-Party tools with your application.
+
+## Tom Select
+
+[Tom Select](https://tom-select.js.org/) is a dynamic, framework agnostic, and lightweight (~16kb gzipped) `<select>` UI control. With autocomplete and native-feeling keyboard navigation, it's useful for tagging, contact lists, country selectors, and so on.
+
+If you application uses Tom Select, you can import the styles from Tom Select along with the Design Systems tokens applied to it.
+```scss
+@import '@rolemodel/rolemodel-design-system/dist/scss/rolemodel-design-system'
```

Comment:
@Jeremy-Walton This assumes they haven't already added `rolemodel-design-system`? If so we should probably clarify with something like this:

> If _**your**_ application uses Tom Select, you can import the styles from Tom Select. _**If you have not already imported the design system you will need to import it to use its related Design Systems tokens.**_

Reply:
Also, typo. `If you...` should be `If your`.

Reply:
Simplified it by removing the import line and just saying `If your application uses Tom Select, you can import the styles from Tom Select along with the Design Systems tokens applied to it. By placing this after importing the base design system.`